src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import './utils/variables';
+@import './utils/mixins';
+@import './blocks/header';
+@import './blocks/main';
+@import './blocks/card';
+@import './blocks/star';
+
 body {
   margin: 0;
+  padding: 20px;
+  font-family: Roboto, Arial, sans-serif;
+  background-color: #e5e5e5;
+
+  --accent-color: #00acdc;
```

Comment:
CSS custom properties (variables) should be defined in the `variables.scss` file for better maintainability, not within the `body` selector.